lstmcpipe/config_MC_prod.yml

```diff
@@ -15,8 +15,11 @@
 workflow_kind: lstchain  # Not expected to be changed by the user
 #prod_id:                 # ex: local_no_n_islands. Default; v00 (if key left empty or None)
 
-base_path_dl0: /fefs/aswg/workspace/MC_data_simlink  # Default user path
+base_path: /fefs/aswg/workspace/MC_data_simlink  # Default user path
   # lstanalyzer user : /fefs/aswg/data/mc
+  #
+# Only for dl1 reprocessing
```

Comment:
If LST insists on it.
For me it feels like some lstchain slang, that will change at some point.
This could just as well do dl1a to dl1a of the lstchain scripts would support it

Reply:
Like dl1ab but you dont discard the images?
I dont think anyone does that, just thinking out loud

Reply:
The images are not necessarily discarded during the dl1ab process.

Reply:
Ah, but then its not dl1ab necessarily, right?

Reply:
> I dont think anyone does that, just thinking out loud

Yes, I would do that anytime for deep learning applications where I want to compare to the RF.
I would never discard the images but sometimes recompute parameters with a different cleaning.

What I mean is that discarding the images or not is not the point here, we want to reprocess the images to recompute parameters.

Reply:
> Ah, but then its not dl1ab necessarily, right?

I don't understand why it would not be. Or we understand differently the meaning of `dl1ab`.

Reply:
So you want the images in the output files as well? So coming from dl1a you want to get to dl1ab ?
I always read the script name as from a to b :sweat_smile:

Reply:
For me 
```
dl1ab = DL1[images + parameters] --> DL1[images + new_parameters]
```

Reply:
Removing the images from the new DL1 file is an option imo. But ok that could be the default if we assume we can easily retrieve the source DL1.

Reply:
I think this shows how much DL1 should actually be splitted in two levels, first creating DL1[images] and then DL1[parameters] as a new level and/or file... but ok we have to deal with CTA nomenclatura.

Reply:
I am just super confused by the name of the script.
So for my understanding:
`dl1ab` in general refers to reprocessing dl1(a) files. Doesnt matter if it writes out dl1a, dl1b or dl1ab, right?

Reply:
So
dl1a -> dl1a would still be called `dl1ab`?
(Of course you usually calculate dl1b as well, but still)

Reply:
> dl1a -> dl1a would still be called dl1ab?

But what would be the meaning of `dl1a -> dl1a` ??
You can't recompute the images from DL1 files.

I think I understand the confusion though.
`dl1ab` always means recomputing the **parameters**.      
But:
- including the original images in the output file has my preference to keep consistent file format and because it's useful in some use cases.
- though I understand it uses disk space and is not necessary in most use cases or if you always can retrieve the original file that contains the images